src/java/fr/paris/lutece/plugins/forms/web/form/panel/display/IFormPanelDisplay.java

```diff
@@ -108,7 +110,9 @@ public interface IFormPanelDisplay extends IFormListPosition
      * 
      * @param locale
      *            The locale used to build the template
+     * @param request
+     *            The HttpServletRequest used to build the template
      * @return the built template of the FormPanelDisplay
      */
-    String buildTemplate( Locale locale );
+    String buildTemplate( HttpServletRequest request, Locale locale );
```

Comment:
Have you assessed the impact of changing the signature of this method on the modules that are dependent on Forms?

Reply:
Yes, the method is only used in one class and is not used in other modules.

Reply:
Nevertheless, it's preferable to pass a map of Strings as parameter instead of the Request object to avoid unnecessary dependencies